src/Epub.ts

```diff
@@ -75,6 +86,10 @@ export default class Epub {
     const navDocStr = await Epub.decryptStr(navDocBuffer, decryptor);
     const navDoc = Epub.parseNavDoc(navDocStr);
 
+    const encryptionPath = fetcher.getEncryptionPath(containerXmlPath);
+    const encryptionStr = await fetcher.getOptionalFileStr(encryptionPath);
+    const encryptionDoc = Epub.parseEncryptionDoc(encryptionStr);
```

Comment:
Could we add a comment here about the these values noting that effectively if any of these methods returns `undefined` the ePub is considered to be unencrypted?